lms/djangoapps/ccx/views.py

```diff
@@ -407,6 +424,47 @@ def ccx_schedule(request, course, ccx=None):  # pylint: disable=unused-argument
     return HttpResponse(json_schedule, mimetype='application/json')
 
 
+def _get_valid_student_email(identifier):
+    """
+    helper function to get an user email
+    from an identifier and validate it
```

Comment:
This docstring isn't super helpful, it doesn't explain what an identifer is, and what values are returned or exceptions raised. See https://github.com/edx/edx-platform/wiki/Python-Guidelines#follow-pep-257